### ISSUE TITLE
corerun does not use CORE_ROOT on Linux

### ISSUE THREAD

**Issue by hadibrais** [dotnet/coreclr#10928]
According to the [documentation](https://github.com/dotnet/coreclr/blob/master/Documentation/workflow/UsingCoreRun.md), corerun is supposed to check whether CORE_ROOT is defined to get the directory in which libcoreclr.so and the other CLR libraries reside. Note that it supports the -c switch for the same purpose.

By going through the source code of the Windows and macOS corerun, it seems that the Windows host checks for CORE_ROOT but the macOS one doesn't.

**Comment by danmosemsft** [comment]
I don't see a reason for that. Do you want to submit a PR, @hadibrais ?

**Comment by hadibrais** [comment]
@danmosemsft Yes, I can fix this. It's just a matter of adding a function call to check whether CORE_ROOT is defined and get the directory from there if it is. I can test the changes on Linux, but I don't have a macOS to test it on.
